README.md

```diff
@@ -87,6 +87,23 @@ Viewing word clouds doesn't require login access. The URL of the view page is in
         font size, the higher the voice pitch.
         - **"Apply conventional weighting"**: When enabled, more frequently submitted words will be highlighted.
 
+### Simulate a Word Cloud
+
+You can find a Node.js script to generate or update existing word cloud answers.
+
+**Steps**
+
+1. **Find a Specific Word Cloud's Question File name**:
+    - In this repository, you can find a specific word cloud's question data file under _data directory; they have suffic of -question.
+2. **Get File Name of the Answers to the Word Cloud**:
+    - From the step 1, you can get the answers file by replacing the -question with -answers; for example, a word cloud that has a question file named xxxx-xxxx-question.json have an answers file named xxxx-xxxx-answers.json.
+3. **Use the Script Located in the Scripts Directory**:
```

Comment:
"scripts" directory -> "src/scripts" directory. This comment also applies to other occurrences. Also add a link for this directory for the users' convenience.